plugins/importexport/onix30/Onix30ExportPlugin.php

```diff
@@ -198,4 +202,29 @@ public function getAppSpecificDeployment($context, $user)
     {
         return new Onix30ExportDeployment($context, $user);
     }
+
+    /**
+     * Helper function to retrieve funding data when available.
+     */
+    public function getFundingData(int $contextId, int $submissionId): Collection|false
+    {
+        if (!PluginRegistry::getPlugin('generic', 'FundingPlugin')) {
+            return false;
+        }
+
+        $fundingData = DB::table('funders AS f')
+            ->select('f.funder_id', 'f.funder_identification', 'fs.setting_value as funder_name', 'fa.funder_award_id', 'fa.funder_award_number')
+            ->where('submission_id', $submissionId)
+            ->where('context_id', $contextId)
```

Comment:
It seems it is not needed to have
f.submission_id and f.context_id,
but maybe to add it however?